app/controllers/IdentityVerification.scala

```diff
@@ -73,7 +74,9 @@ class IdentityVerification @Inject()(
         case true =>
           identityVerificationService.continue(journeyId, request.userDetails).map {
             case Some(RegistrationSuccess(personId)) =>
-              Redirect(controllers.registration.routes.RegistrationController.success(personId))
+              if (config.newRegistrationJourneyEnabled)
+                Redirect(controllers.registration.routes.RegistrationController.confirmation(personId))
+              else Redirect(controllers.registration.routes.RegistrationController.success(personId))
```

Comment:
can we use an import to avoid fully-qualified names

Reply:
Controller are in mutiple packages and this IdentityVerification.scala using reference to different controllers routes so needed to have qualified names

Reply:
I think you can remove the unused import on line 21 and do 
```
if (config.newRegistrationJourneyEnabled)
                Redirect(registration.routes.RegistrationController.confirmation(personId))
              else Redirect(registration.routes.RegistrationController.success(personId))
```
that would be a little better

and same on line 64

I find `routes` quite tricky tbh